src/settings.py

```diff
@@ -93,6 +93,39 @@ def save_splanes_to_file(splanes, filename):
         for p in splanes:
             out.write(p.toFile() + '\n')
 
+def get_version(settings_filename):
+    try:
+        with open(settings_filename, "r") as settings_file:
+            settings = yaml.safe_load(settings_file)
+
+        version = settings["common"]["version"]
+        return version
+    except Exception as e:
+        print("Error reading version")
+        return ""
+
+def paths_transfer_in_settings(initial_settings_filename, final_settings_filename):
+    with open(initial_settings_filename, "r") as settings_file:
+        initial_settings = yaml.safe_load(settings_file)
```

Comment:
here you should update all settings, not only the paths. 
for k,v := in old_settings {
   new_settings[k]=v
}